.semgrep/rxjs-syntax.yaml

```diff
@@ -0,0 +1,24 @@
+rules:
+
+  - id: combineLatest-not-combining
+    pattern: combineLatest([$SINGLE_EXPRESSION])
+    fix:  $SINGLE_EXPRESSION
+    message: combineLatest is not needed with a single argument
+    languages: [ts]
+    severity: WARNING
+
+  - id: observable-not-ending-with-dollar-sign
```

Comment:
This rule was rewritten after the file was moved, which is why it does not just show up as a rename.